docker/service/src/main/java/uk/nhs/adaptors/scr/services/ACSService.java

```diff
@@ -0,0 +1,41 @@
+package uk.nhs.adaptors.scr.services;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.stereotype.Component;
+
+import com.github.mustachejava.DefaultMustacheFactory;
+import com.github.mustachejava.Mustache;
+import com.github.mustachejava.MustacheFactory;
+
+import uk.nhs.adaptors.scr.models.ACSPayload;
+
+@Component
+public class ACSService {
+
+    private static final String SET_RESOURCE_PERMISSIONS_TEMPLATE_PATH = "set_resource_permissions_template.mustache";
+
+    public String handleMessage(ACSPayload acsSetResourceObject) {
+
+        MustacheFactory mf = new DefaultMustacheFactory();
+        Mustache m = mf.compile(SET_RESOURCE_PERMISSIONS_TEMPLATE_PATH);
```

Comment:
As template is a constant - maybe it's worth extracting it's compilation to when class is loaded/created? Dping that on every message is probably rather heavy.